src/legacy/core_plugins/data/public/filter/filter_bar/filter_editor/_filter_editor.scss

```
@@ -0,0 +1,3 @@
+.filter-editor__field {
+  max-width: 208px;
```

Let's use EUI variables with calculations here. This would equate to `$euiSize * 13`
```suggestion
  max-width: $euiSize * 13;
```

---

Nice tip! I really need to learn more about the Eui project (although I have already submited some PRs to the project...)

---

No problem! There's a whole page on SASS in the EUI docs that would be a good reference: https://elastic.github.io/eui/#/guidelines/sass